### ISSUE TITLE
floatingLabel glitch

### ISSUE THREAD

**Issue by theapache64** [GeekyAnts/NativeBase#2000]
Input with floatingLabel takes sometime to float the label.

### Code
```
import React, {Component} from 'react';
import {Container, Content, Form, Input, Item, Label} from "native-base";

export default class LabScreen extends Component {
    render() {
        return (
            <Container>
                <Content>
                    <Form>
                        <Item floatingLabel>
                            <Label>Sample label</Label>
                            <Input
                            />
                        </Item>
                    </Form>
                </Content>
            </Container>
        );
    }
}

```
### Output

![untitled](https://user-images.githubusercontent.com/9678279/41301419-8e92a5a6-6e85-11e8-943e-87b725376b7c.gif)

### I am on
```
"native-base": "^2.5.1",
    "react": "16.3.1",
    "react-native": "0.55.4",
```

**Comment by SupriyaKalghatgi** [comment]
Share the device configuration
As in, model of device, which OS, which API etc

**Comment by ZeroCool00** [comment]
same problem.. i have a nougat
![whatsapp image 2018-06-13 at 12 09 19 pm](https://user-images.githubusercontent.com/18139594/41334280-abcfe8ae-6f02-11e8-9728-3898619c584f.jpeg)

**Comment by theapache64** [comment]
@SupriyaKalghatgi 

![screenshot_2018-06-13-12-58-59](https://user-images.githubusercontent.com/9678279/41336467-f964b44e-6f09-11e8-952f-117def3dc8af.png)

**Comment by theapache64** [comment]
I've also tested it in my emulator which runs Android 7. Same issue. I think it's not device/os specific.

**Comment by SupriyaKalghatgi** [comment]
we dd'nt face it ever
hence asked you for more details

**Comment by ZeroCool00** [comment]
like i said.. it never happen before.. its just happen in this version **native-base": "^2.5.1**

**Comment by akhil-geekyants** [comment]
@theapache64 @ZeroCool00 can you try toggling the remote debugging mode.

**Comment by theapache64** [comment]
@akhil-geekyants Yeah, Turning off the debugging mode helped. (y)

**Comment by akhil-geekyants** [comment]
Attaching a gif with `native-base` version 2.5.1

![floatinglabel](https://user-images.githubusercontent.com/27768184/41353665-8393a03c-6f3a-11e8-8b82-3a0cdf561055.gif)

**Comment by theapache64** [comment]
It worked when i disabled the debug mode

**Comment by ViktorAksionov** [comment]
Bug still exist even in production version!

**Comment by akhil-geekyants** [comment]
@ViktorAksionov can you check this against branch `revert/fix-1606`. Let us know if it is fixed.

**Comment by ViktorAksionov** [comment]
@akhil-geekyants 
This patch fixed my problem

**Comment by SupriyaKalghatgi** [comment]
Fixed with [2.6.1](https://github.com/GeekyAnts/NativeBase/releases/tag/v2.6.1)
